translator/trace/zipkinv2/protobuf.go

```diff
@@ -0,0 +1,67 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package zipkinv2
+
+import (
+	zipkinmodel "github.com/openzipkin/zipkin-go/model"
+	"github.com/openzipkin/zipkin-go/proto/zipkin_proto3"
+
+	"go.opentelemetry.io/collector/internal/model"
+)
+
+var _ model.TracesDecoder = (*protobufDecoder)(nil)
+
+type protobufDecoder struct {
+	// DebugWasSet toggles the Debug field of each Span. It is usually set to true if
+	// the "X-B3-Flags" header is set to 1 on the request.
+	DebugWasSet bool
```

Comment:
Hah, very cryptic name but I agree that we should use what zipkin is using.

Reply:
Also why public in a private structure?